terraform/deployments/apps/test/publisher/main.tf

```diff
@@ -16,6 +16,10 @@ terraform {
 
 provider "aws" {
   region = "eu-west-1"
+
+  assume_role {
+    role_arn = var.assume_role_arn
+  }
```

Comment:
Probably you've already checked this, but is it possible to only pass an `assume_role` block to the root module provider?

Reply:
I didn't actually check - it would make it _much_ simpler if we only specified the provider's config at the deployment level (and didn't pass the assume_role_arn into the modules)